worldedit_commands/init.lua

```diff
@@ -456,15 +456,35 @@ minetest.register_chatcommand("/replaceinverse", {
 })
 
 local check_sphere = function(name, param)
-	if worldedit.pos1[name] == nil then
-		worldedit.player_notify(name, "no position 1 selected")
-		return nil
-	end
 	local found, _, radius, nodename = param:find("^(%d+)%s+(.+)$")
 	if found == nil then
 		worldedit.player_notify(name, "invalid usage: " .. param)
 		return nil
 	end
+	if nil ~= area_protection.areas then

```

Comment:
I'm not sure yet how to best handle this. I was considering whether it may become important to support different area/node protection mods. So, semantically, the area_protection.areas slot is not only meant as an accessor for the areas mod API, but also as a conditional saying that the areas mod is the currently applicable area/node protection interface.